scripts/startAppForCypress.js

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
```

Comment:
Is this something other apps using as well, what is the problem is solving?

Maybe we could write a small comment on the top explaining that for future devs.